webapp/src/pages/EditAvatarPage/EditAvatarPage.tsx

```diff
@@ -58,18 +67,32 @@ export default function EditAvatarPage() {
           avatarY: FormatNumber(picturePosition.y),
```

Comment:
Maybe we should do this transformation inside useDrag, passing FormatNumber as a callback as well, or maybe we shouldn't pass reverseTransform as a callback here, and do both transformations here. This is something I was thinking of doing. What do you think?

Reply:
I think just passing the transformed values to `useDrag` would be the best :+1: